### PR TITLE
circe 0.11

### PR DESCRIPTION
**Pull request by tomaszym**
Hello,
I would be very grateful if this was realeased ASAP.  :pray:  

BTW you might want to release `1.2.2` with circe `0.9.3` too.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -18,10 +18,10 @@ scalacOptions ++= {
     Seq("-target:jvm-1.7")
 }
 
-val circeVersion = "0.9.3"
+val circeVersion = "0.10.0"
```
Comment (sullis) [inline build.sbt]: please update to 0.10.1

Comment (tomaszym) [reply build.sbt]: done!

**Comment by poohsen** [comment]
+1 to merging this!

**Comment by travisbrown** [comment]
Superseded by https://github.com/sangria-graphql-org/sangria-circe/pull/1 and can be closed.

**Pull request closed without merging**